ricecooker/classes/questions.py

```diff
@@ -238,6 +236,38 @@ def process_image_field(self, data, downloader):
             new_data[new_key] = new_data.pop(k)
         return new_data, files
 
```

Comment:
May be able to delete the method below; might be residual from one of the many merge conflicts, and no longer needed.